raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala

```diff
@@ -205,6 +221,29 @@ object Rasterizer {
     * LineString.  The iteration happens in the direction from the
     * first point to the last point.
     */
+  def foreachCellByLineString(
+    line: Line,
+    re: RasterExtent,
+    options: Options
+  )(f: (Int, Int) => Unit) {
+    val cells = (for(coord <- line.jtsGeom.getCoordinates()) yield {
+      (re.mapXToGrid(coord.x), re.mapYToGrid(coord.y))
+    }).toList
+
+    for(i <- 1 until cells.size) {
+      foreachCellInGridLine(
+        cells(i - 1)._1, cells(i - 1)._2,
```

Comment:
I'll confess that I did not look very closely at this code, I just copied what was there.